pkg/cmdutil/util.go

```diff
@@ -0,0 +1,31 @@
+package cmdutil
+
+import "strings"
+
+func GetNestedStringField(inputMap map[string]interface{}, selector string) (string, bool) {
+	selectors := strings.Split(selector, ".")
+	currentMap := inputMap
+	for index, key := range selectors {
+		if value, ok := currentMap[key]; ok {
+			if index+1 == len(selectors) {
+				result, isResultString := value.(string)
+				return result, isResultString
+			} else if mapVal, isMapVal := value.(map[string]interface{}); isMapVal {
+				currentMap = mapVal
+			} else {
+				return "", false
```

Comment:
Is this a case where we cannot parse the node of the JSON object?  Do we want to differentiate between "not found" and failure to parse?  (If we did fail to parse, we would probably want to just `panic`...

For JSON, there are two types of structures we need to be able to parse: 
* An array of things, `[]interface{}`
* A map of things, `map[string]interface{}`

Reply:
and I guess maybe we are simplifying and not handling arrays?

(This would probably be a great candidate for some unit tests.)